tests/e2e/docusaurus2-graphql-doc-generator-multi-instance-2.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  id: "config-B",
+  schema: "data/tweet-B.graphql",
+  rootPath: "./docs-tweet-B",
```

Comment:
I'd rather limit the number of local schemas to maintain
```suggestion
  id: "swapi",
  schema: "data/swapi.graphql",
  rootPath: "./doc-swapi",
```

Reply:
Addressed in 9b550e2.